source/features/github-bugs.css

```diff
@@ -13,11 +13,6 @@
 	margin-left: 8px;
 }
 
-/* Avoid scroll bar on single-line `pre` in Markdown #4899 */
```

Comment:
No longer reproducible in https://github.com/refined-github/github-url-detection#install

<img width="851" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/183243797-6a1b9299-8c4f-4b96-8dd2-2849f81aed6f.png">